src/main/js/LogoutPage.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import Header from 'component/common/Header';
+import { connect } from "react-redux";
+
+class LogoutPage extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div className="wrapper">
+                <div className="loginContainer">
+                    <div className="loginBox">
+                        <Header />
+                        <div className="form-horizontal loginForm">
+                            <div className="row">
+                                <div className="col-sm-12 text-center">
+                                    <div className="d-inline-flex flex-column p-2">You've successfully logged out of Alert!</div>
```

Comment:
Do we need to escape the ' character here?

Reply:
I will double check it looked fine in the UI because it is in a JSX section the raw html will be interpreted as html.  I will check though